scipy/stats/_continuous_distns.py

```diff
@@ -1828,6 +1829,157 @@ def _stats(self, a):
 dgamma = dgamma_gen(name='dgamma')
 
 
+class dpareto_lognorm_gen(rv_continuous):
+    r"""A double Pareto lognormal continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `dpareto_lognorm` is:
+
+    .. math::
+
+        f(x, \mu, \sigma, \alpha, \beta) =
+        \frac{\alpha \beta}{(\alpha + \beta) x}
+        \phi\left( \frac{\log x - \mu}{\sigma} \right)
+        \left( R(y_1) + R(y_2) \right)
+
+    where :math:`R(t) = \frac{1 - \Phi(t)}{\phi(t)}` is a Mills' ratio,
+    :math:`y_1 = \alpha \sigma - \frac{\log x - \mu}{\sigma}`,
+    and :math:`y_2 = \beta \sigma + \frac{\log x - \mu}{\sigma}`
+    for real numbers :math:`x` and :math:`\mu`, :math:`\sigma > 0`,
+    :math:`\alpha > 0`, and :math:`\beta > 0` [1]_.
```

Comment:
For the sake of completeness, let's also explain $\Phi$ and $\phi$. At least I did not know the term Mills' ratio before, so maybe add the Wikipedia reference for that one too.

Reply:
I think I'll just remove the term "Mills' ratio. I can mention what $\phi$ and $\Phi$ are; I just didn't before because it's pretty standard in context so many other distributions rely on the fact that all the symbols are defined in the [tutorial](https://docs.scipy.org/doc/scipy/tutorial/stats/continuous.html#continuous-random-variables).